scripts/ci/pylint_todo.txt

```diff
@@ -24,6 +24,7 @@
 ./airflow/models/taskinstance.py
 ./airflow/models/variable.py
 ./airflow/models/xcom.py
+./airflow/models/xcom_arg.py
```

Comment:
As it is a new file let's make it Pylint compatible please

Reply:
We can't... circular imports in core.

Reply:
We just did :) i.e just for now add ignore to the circular dependency in the import. That would still allow Pylint to carry out other tests on that file. If we disable pylint on that file it wouldn't even check for other errors

Reply:
> If we disable pylint on that file it wouldn't even check for other errors

That's true. And I forgot about this trick: 
https://github.com/apache/airflow/blob/master/airflow/__init__.py#L48-L56

Reply:
@jonathanshir can we try to address this problem?